content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -15,68 +15,91 @@ further_reading:
-</div>
+### .NET runtimes
+Datadog .NET Tracer supports instrumentation for .NET Core & [NET Framework][4]. 
 
-## Integrations
+| Version                         | Vendor End of Life |
+| ------------------------------- | ------------------ |
+| .NET 5                          |                    |
+| .NET Core 3.1                   | 12/03/2022         |
+| .NET Core 3.0                   | 03/03/2020         |
+| .NET Core 2.2                   | 12/23/2019         |
+| .NET Core 2.1 (LTS)             | 08/21/2021         |
 
-The .NET Tracer can instrument the following libraries automatically:
-
-| Framework or library            | NuGet package                                                           | Integration Name     |
-|---------------------------------|-------------------------------------------------------------------------|----------------------|
-| ASP.NET Core                    | `Microsoft.AspNetCore`</br>`Microsoft.AspNetCore.App`</br>2.0+ and 3.0+ | `AspNetCore`         |
-| ADO.NET                         | `System.Data.Common`</br>`System.Data.SqlClient` 4.0+                   | `AdoNet`             |
-| HttpClient / HttpMessageHandler | `System.Net.Http` 4.0+                                                  | `HttpMessageHandler` |
-| WebClient / WebRequest          | `System.Net.Requests` 4.0+                                              | `WebRequest`         |
-| Redis (StackExchange client)    | `StackExchange.Redis` 1.0.187+                                          | `StackExchangeRedis` |
-| Redis (ServiceStack client)     | `ServiceStack.Redis` 4.0.48+                                            | `ServiceStackRedis`  |
-| Elasticsearch                   | `Elasticsearch.Net` 5.3.0+                                              | `ElasticsearchNet`   |
-| MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
-| PostgreSQL                      | `Npgsql` 4.0+                                                           | `AdoNet`             |
-| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                | `RabbitMQ`           |
-| Service Fabric Remoting         | `Microsoft.ServiceFabric.Services.Remoting` 4.0.470+                    | `ServiceRemoting`    |
-
-<div class="alert alert-info">
-<strong>Note:</strong> The ADO.NET integration instruments calls made through the <code>DbCommand</code> abstract class or the <code>IDbCommand</code> interface, regardless of the underlying implementation. It also instruments direct calls to <code>SqlCommand</code>.
-</div>
+ Additional information on .NET Core support policy can be found at [Microsoft .NET Core Lifecyle Policy][5]. 
+
+### Supported Process Architecture
+
+| Processor architectures                                                                                            |
```

Comment:
I think this could be improved by having two columns, something like this:

```
| Processor architecture           | First Supported Runtime |
| ------------------------------- | ------------------ |
| Windows x86                         | .NET Core 2.1      |
| Windows x64                         | .NET Core 2.1      |
| Linux x64                               | .NET Core 2.1      |
| Alpine Linux x64                    | .NET Core 2.1      |
| Linux ARM64                         | .NET 5      |
```

Reply:
I thought about this but I didn't want it to be preserved as a specific version only works for a specific process architecture.

Reply:
Yeah that makes sense, resolving